### PR TITLE
Add Option For Exponent Base in Priority Satisfaction Metric

### PR DESCRIPTION
**Pull request by Hedgemon4**
### Description
- added an option for the exponent base in the priority satisfaction metric

### CODE REVIEW

File: benchmarking/evaluations/metrics/priority_satisfaction.py [hunk 1]
```diff
@@ -11,10 +11,18 @@ class PrioritySatisfaction(TeamSetMetric):
     Calculates the number of priorities satisfied by the algorithm
     """
 
-    def __init__(self, priorities: List[Priority], is_linear: bool, *args, **kwargs):
+    def __init__(
+        self,
+        priorities: List[Priority],
+        is_linear: bool,
+        exponent_base: int = 2,
```
Comment (Opeyem1a) [inline benchmarking/evaluations/metrics/priority_satisfaction.py:18]: nit;

uhhhh, how does this affect existing code? Feel like it should default to whatever the actual prio algorithm is using, like import NUM_BRACKETS

**Review by Opeyem1a — APPROVED** [review]

File: benchmarking/evaluations/metrics/priority_satisfaction.py (hunk 1 above)
Comment (Hedgemon4) [reply benchmarking/evaluations/metrics/priority_satisfaction.py:18]: This is what it was before. It was not the same as num buckets previously.

Comment (Hedgemon4) [reply benchmarking/evaluations/metrics/priority_satisfaction.py:18]: Let me know if this is fine

Comment (Opeyem1a) [reply benchmarking/evaluations/metrics/priority_satisfaction.py:18]: O

**Review by Opeyem1a — APPROVED** [review]

**Pull request merged**